### DISCUSSION TITLE
Best way to add "target" to LinkField

### DISCUSSION THREAD

**Discussion by devlife** [OrchardCMS/OrchardCore#11329] [Q&A]
What's the best way to add a `Target` field to a `LinkField`? I tried using a `ShapeTableProvider` but am having issues identifying the correct location (ie. `Content:10` ) to get it to display next to/under the existing `LinkField` text boxes in admin.

**Comment by hishamco** [comment]
I'm not sure why we didn't add a `Target` as property into `LinkField`?!!

**Reply by devlife** [reply]
Haha right? Pretty much every project I end up solving it differently. It sounds like perhaps I should just submit a PR to add `Target` to `LinkField` but I would still like to know, in general, the best approach for solving the problem.
